static/src/js/components/CollectionDetails/CollectionDetailsBody.js

```diff
@@ -123,6 +129,7 @@ export const CollectionDetailsBody = ({
     hasAllMetadata,
     gibsLayers,
     nativeDataFormats,
+    relatedCollections = {},
```

Comment:
Did you try a collection that has no related collections? I want to make sure its not returning null here which would ignore the default.